instrumentation/netty/netty-3.8/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/netty/v3_8/NettyChannelInstrumentation.java

```diff
@@ -56,55 +56,55 @@ public void transform(TypeTransformer transformer) {
   public static class ChannelConnectAdvice {
 
     @Advice.OnMethodEnter(suppress = Throwable.class)
-    public static void onEnter(
-        @Advice.This Channel channel,
-        @Advice.Argument(0) SocketAddress remoteAddress,
-        @Advice.Local("otelParentContext") Context parentContext,
-        @Advice.Local("otelRequest") NettyConnectionRequest request,
-        @Advice.Local("otelTimer") Timer timer) {
-
-      parentContext = Java8BytecodeBridge.currentContext();
+    public static Object onEnter(
+        @Advice.This Channel channel, @Advice.Argument(0) SocketAddress remoteAddress) {
+
+      Context parentContext = Java8BytecodeBridge.currentContext();
       Span span = Java8BytecodeBridge.spanFromContext(parentContext);
       if (!span.getSpanContext().isValid()) {
-        return;
+        return null;
```

Comment:
[for reviewer] here the `null` value is used to indicate early exit of the method advice, in the exit advice this is guarded through the `instanceof` condition.